cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -173,10 +205,18 @@ func (dm *deviceManager) PreStartContainer(ctx context.Context, rqt *pluginapi.P
 }
 
 func main() {
+	var mode string
+
+	flag.StringVar(&mode, "mode", "region", "device plugin mode: 'region' (default) or 'af'")
 	flag.Parse()
-	fmt.Println("FPGA device plugin started")
 
-	devs, err := discoverFPGAs(sysfsDirectory, devfsDirectory)
+	if mode != "region" && mode != "af" {
```

Comment:
would it be better to call af mode "function" just for the sake of consistency with "region"?

Reply:
IMHO in the context of FPGAs `function` is too loaded word. `intel-fpga-port.0` can be treated as a virtual function and as a accelerator function. I'd like to emphasize the latter meaning. And `region` is specific enough to point to a unit of reconfiguration.

Reply:
ok, makes sense. How about afu then? Plugin announces afu ids in this mode, right?